src/main/java/in/org/projecteka/hiu/ConsentManagerServiceProperties.java

```diff
@@ -7,11 +7,20 @@
 import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.context.annotation.Configuration;
 
+import java.util.Arrays;
+import java.util.List;
+
+
 @Configuration
 @ConfigurationProperties(prefix = "hiu.consentmanager")
 @Data
 @NoArgsConstructor(access = AccessLevel.PACKAGE)
 @AllArgsConstructor
 public class ConsentManagerServiceProperties {
     private String url;
+    private String suffixes;
+
+    public List<String> getSuffixes(){
+       return Arrays.asList(suffixes.split(" "));
+    }
```

Comment:
One URL can have multiple suffixes?

Reply:
No. But calling different urls based on suffix is not in the scope of the story. So I just added suffix and let the url same.

Reply:
But this doesn't seem right to me as well.